### PR TITLE
Minor changes indicated by code inspection

### PR DESCRIPTION
**Pull request by geoand**
This PR contains super small changes as those where indicated by the code inspection tool I ran

### CODE REVIEW

**Comment by spencergibb** [comment]
@geoand Have you signed the [contributor agreement](https://support.springsource.com/spring_committer_signup)?

File: spring-cloud-consul-bus/src/main/java/org/springframework/cloud/consul/bus/EventService.java [hunk 1]
```diff
@@ -93,8 +93,7 @@ public BigInteger toIndex(String eventId) {
 		String upper = eventId.substring(19, 23) + eventId.substring(24, 36);
 		BigInteger lowVal = new BigInteger(lower, 16);
 		BigInteger highVal = new BigInteger(upper, 16);
-		BigInteger index = lowVal.xor(highVal);
-		return index;
+        return lowVal.xor(highVal);
```
Comment (spencergibb) [inline spring-cloud-consul-bus/src/main/java/org/springframework/cloud/consul/bus/EventService.java]: We use tabs, not spaces.

**Comment by geoand** [comment]
@spencergibb Yes I signed the agreement about 10h ago.
Sorry for the tab/space issue!

**Comment by spencergibb** [comment]
Closed via 3fb8dc07b73fd0be33f49ac8df8d8a58a9bd8194

**Pull request closed without merging**